index.html

```diff
@@ -1,92 +1,106 @@
-  <meta property="og:locale" content="en_US" />
-  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <meta property="og:url" content="https://alasa-ala.pages.dev" />
+  <meta property="og:locale" content="tp_MP" />
+  <meta property="og:image" content="https://alasa-ala.pages.dev/favicon.png"/>
   <link rel="icon" href="/favicon.png" type="image/png" />
   <link rel="stylesheet" type="text/css" href="/style.css">
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 </head>
 <body>
   <main>
-    <h1>Don't ask to ask, just ask</h1>
+    <h1>o toki alasa ala tan toki alasa a.</h1>
     <p>
-      Every now and then, in online chat rooms I hang around in, someone pops
-      in and says something in the lines of,
+      tenpo ale lili la, mi lon toki linluwi. taso, jan li kama li toki e ni:
+      <!--Every now and then, in online chat rooms I hang around in, someone pops
+      in and says something in the lines of,-->
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        Any Java experts around?
+        jan sona Java li lon anu seme?
+        <!--Any Java experts around?-->
       </p>
     </blockquote>
     <p>
-      This is bad form, for several reasons. What the person is
-      <em>actually</em> asking here is,
+      ni li ike wawa tan <em>jan li toki alasa e ni</em>:
+      <!--This is bad form, for several reasons. What the person is
+      <em>actually</em> asking here is,-->
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        Any Java experts around who are willing to commit into looking into my
+        jan sona Java lon ma ni li, wile e pona e nimi Java mi? taso Java ala la jan ale li ken ala ken pana e sona tawa mi?
+        <!--Any Java experts around who are willing to commit into looking into my
         problem, whatever that may turn out to be, even if it's not actually
         related to Java or if someone who doesn't know anything about Java
-        could actually answer my question?
+        could actually answer my question?-->
       </p>
     </blockquote>
     <p>
-      There are plenty of reasons why people who DO have the knowledge would
+      jan pi jo sona li pana e sona ala, tan nasin sona mute. tenpo la, sina toki alasa e sona ete.
+      <!--There are plenty of reasons why people who DO have the knowledge would
       not admit to it. By asking, you're asking for more than what you think
-      you're asking.
+      you're asking.-->
     </p>
     <p>
-      You're asking people to take responsibility. You're questioning people's
+      sina wile e jan ete li lawa sona. sina toki alasa e sona jan ete. sina pake e jan ete li pana e sona kin. tenpo mute la, mi pana e sona tawa jan mute, tan nasin pali li sama sama tawa jan nimi linluwi.
```

Comment:
preferable, avoid using words like ete and linluwi- ete is only used by 5% of the toki pona community, and linluwi is only used by 29%

Reply:
tried to make it shorter. the changed one is like three times longer than english ._.